examples.js

```diff
@@ -0,0 +1,20 @@
+const path = require('path');
+const app = require('connect')();
+const winston = require('winston');
+
+process.env.NODE_ENV = 'development';
```

Comment:
todo esto debería ir en una carpeta examples/ serado del source code del package.

Reply:
En realidad, en el submodule: https://github.com/indec-it/react-form-builder-example